src/Robot.cpp

```diff
@@ -51,7 +52,7 @@ void Robot::DisabledPeriodic() {
   // Scheduler must start running before doing any operations
   // on subsystems or commands.
   Shooter::getInstance()->GetSetShooterCommand()->SetSpeed(0.0);
-}
+ }
```

Comment:
This change isn't needed.

I believe that `Robot.cpp` should not need any changes for this work.